src/lib.rs

```diff
@@ -197,10 +221,22 @@ impl View for FlexiLoggerView {
         // The longest line sets the width
         let w = logs
             .iter()
-            .map(|msg| msg.width())
+            .map(|msg|
+                msg.spans().map(|x|
+                    // if the log message contains more than one line,
+                    // only the longest line should be considered
+                    // (definitely not the total content.len())
+                    x.content.split('\n').map(|x| x.width()).max().unwrap()
+                ).sum::<usize>()
```

Comment:
Wait - do you really want the _sum_ of the longest line of each span?
For example, if you have 3 spans:
* `"a b c"`
* `" d\ne f"`
* `"g"`
Then the text would look like:
```
a b c d
e f g
```
And the width should be the maximum line width _across_ spans: the width of `"a b c d"`, not the width of `"a b c"` + the width of `"e f"` + the width of `"g"`.